core/src/main/java/pl/allegro/tech/embeddedelasticsearch/ElasticSearchInstaller.java

```diff
@@ -88,6 +94,14 @@ private void installPlugins() throws IOException, InterruptedException {
         }
     }
 
+    private static boolean isPluginInstalled(InstallationDescription.Plugin plugin, String[] pluginList) {
```

Comment:
Once you have non-null set of plugin names, this method will reduce to checking if set contains plugin name